test/suite/test_timestamp27.py

```diff
@@ -105,3 +105,10 @@ def test_rollback_timestamp_with_commit_timestamp(self):
         self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda:
         self.session.timestamp_transaction('rollback_timestamp=' + self.timestamp_str(100) + ",durable_timestamp=" + self.timestamp_str(100)),
             '/commit timestamp and rollback timestamp should not be set together/')
+
+    def test_prepare_timestamp_before_stable_timestamp(self):
+        self.session.begin_transaction("roundup_timestamps=(prepare=true)")
+        self.conn.set_timestamp('stable_timestamp=' + self.timestamp_str(100))
+        self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda:
+        self.session.prepare_transaction('prepare_timestamp=' + self.timestamp_str(50)),
```

Comment:
The `txn_config` function was called as part of the `begin transaction`. The error should occur as part of the begin? Am I missing anything?

Reply:
Yes, that's correct. Fixing now